components/datetime/src/neo_skeleton.rs

```diff
@@ -73,6 +73,33 @@ impl NeoSkeletonLength {
     }
 }
 
+/// A specification for when to display the era when formatting a year.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "serde", serde(rename_all = "lowercase"))]
+#[non_exhaustive]
+pub enum EraDisplay {
+    /// Always display the era.
+    ///
+    /// Examples:
+    ///
+    /// - `1000 BC`
+    /// - `77 AD`
+    /// - `2024 AD`
+    Always,
+    /// Display the era when needed to disambiguate the year.
```

Comment:
document this better. iiuc it's for years 0..1000, is that guaranteed?

Reply:
I'll come back and update this spec text once a decision is reached on the era display design in https://docs.google.com/document/d/1dmMk_XODm3DGe84GmMVw7O6a7oIs5yojDXm_25CcMbw/edit